libs/openchallenges/api-description/src/components/schemas/CreationDatetime.yaml

```diff
@@ -0,0 +1,4 @@
+description: Datetime when metadata was added to the OC database.
```

Comment:
A more reusable description would be (to apply to `CreatedDatetime`):

> Datetime when this object was added to the database.

Similar comment probably applies to `ModifiedDatetime`

Reply:
I agree. I ideally we should have both `CreatedDatetime` (when it's initially  added to the data) and `ModifiedDatetime`. By the way, do `CreatedDateTime` looks better than `CreatedDatetime` to you? 

I would prefer @vpchung to alter the metadata spreadsheet next year~ I'm afraid to break something in data :).

Reply:
> By the way, do CreatedDateTime look better than CreatedDatetime to you?

Yes. Also we could rename `ModifiedDatetime` to `UpdatedDateTime` since we use it for the property `updatedAt`.

Reply:
Done